### ISSUE TITLE
Cannot select Mixed Reality mode on SteamVR with Vive 1

### ISSUE THREAD

**Issue by GyrosGeier** [jrbudda/Vivecraft_116#135]
I haven't been playing for a while, and just upgraded from 1.12 to 1.19, but cycling through the list of mirror modes with F5 does not offer Mixed Reality mode. I did play with MR view on stream in 1.12, so it worked before.

The setup is fairly standard, with a GTX980, Vive 1 and SteamVR. LIV has been installed for a while, and there are two extra controllers registered that are offline (because LIV is not running in the background when I'm running Minecraft).

The launcher log file mentions that a "Mixed Reality" texture has been created, with the same resolution as each of the eyes.

**Comment by fayer3** [comment]
it should work with 1.16 iirc, it only should be missing in 1.17+

**Comment by Techjar** [comment]
Yeah it's disabled for now as it's broken. We haven't yet figured out how to fix its shader with the new rendering pipeline.

**Comment by GyrosGeier** [comment]
Eep, that's bad because I'd like to join a multiplayer game, so I don't have much choice in version.

Can this be documented on the homepage?

Is there a description of the problem with the shader, in case I (or someone else) wants to take a look at it?
